src/marine/cryosat_ice2ioda.py

```diff
@@ -75,9 +73,9 @@ def _read(self):
             for i in range(len(lons)):
                 obs_date = reftime + timedelta(seconds=float(time[i]))
                 locKey = lats[i], lons[i], obs_date.strftime("%Y-%m-%dT%H:%M:%SZ")
-                self.data[0][locKey][valKey] = vals[i]
-                self.data[0][locKey][errKey] = 0.1
-                self.data[0][locKey][qcKey] = qc[i]
+                self.data[locKey][valKey] = vals[i]
+                self.data[locKey][errKey] = 0.1
+                self.data[locKey][qcKey] = qc[i]
 
             os.remove("cryosat_nc4classic.nc")
```

Comment:
Here, too.

Reply:
This one we kept. Since we don't need to keep the intermediate file: cryosat_nc4classic.nc

Reply:
Can we leave it as is for now?

Reply:
@CoryMartin-NOAA are you okay with leaving the code as is? If so, I will merge (as long as the tests pass of course). Thanks!

Reply:
@srherbener sure it's fine especially since these lines existed before this PR